src/components/DateInput.jsx

```diff
@@ -126,15 +127,20 @@ class DateInput extends React.Component {
 
   onKeyDown(e) {
     e.stopPropagation();
+    if (!['Shift', 'Control', 'Alt', 'Meta'].includes(e.key)) {
```

Comment:
for perf, this should probably be an object literal or a Set defined at the module level.

Reply:
Fixed.